com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -1843,7 +1843,7 @@ internal void MoveObjectsToDontDestroyOnLoad()
         /// Using the local scene relative Scene.handle as a sub-key to the root dictionary allows us to
         /// distinguish between duplicate in-scene placed NetworkObjects
         /// </summary>
-        private void PopulateScenePlacedObjects(Scene sceneToFilterBy, bool clearScenePlacedObjects = true)
+        internal void PopulateScenePlacedObjects(Scene sceneToFilterBy, bool clearScenePlacedObjects = true)
```

Comment:
Do we really need to use the InternalsVisibleTo hack to test this? What we care about is that in a Stop->Start scenario scene-placed objects work, right? Can we not test that scenario just using public APIs?

Reply:
For right now, yeah that is the only way to test this fix.
However, there is currently an effort to look into fixing in-scene NetworkObjects (in general) that would allow me to write a proper integration test... but that is a future PR.